nubis/bin/migrate

```diff
@@ -1,34 +1,119 @@
 #!/bin/bash
+#
+# This script is run on every ami change.
+#+ This is the place to do things like database initilizations and migrations.
```

Comment:
Yes, it is an old commenting style that... well i honestly cant remember where it comes from any more :)